### ISSUE TITLE
@import throws off indent rule starting with v1.8.0

### ISSUE THREAD

**Issue by cappadona** [sasstools/sass-lint#779]
Upgraded to `v1.8.2` earlier today and encountered some funkiness with indentation. Tracked it down to being introduced with `v1.8.0`.

Here's a sample file to reproduce this issue (using the default config):

**indent-fail.scss**
```scss
@import 'anything';

// Colors variable via accoutrement-color
$colors: (
  'white': hsl(0, 0%, 100%),
  'black': hsl(0, 0%, 0%),
  'aluminum': hsl(193, 6%, 68%),
  'light-gray': 'aluminum' ('tint': 40%),
  'bunker': hsl(210, 6%, 7%),

  // Palette
  'malta': hsl(33, 21%, 69%),
  'soapstone': hsl(39, 30%, 89%),
);
```

Here's the CLI output when using `v1.8.2` (or anything >= `v1.8.0`):

```sh
$ sass-lint indent-fail.scss -v

indent-fail.scss
   9:3  warning  Expected indentation of 6 spaces but found 2  indentation
  11:3  warning  Expected indentation of 6 spaces but found 2  indentation
  12:3  warning  Expected indentation of 6 spaces but found 2  indentation
  13:3  warning  Expected indentation of 6 spaces but found 2  indentation

✖ 4 problems (0 errors, 4 warnings)
```

here's a diff with a sample file that doesn't exhibit this issue (comment out the the `@import`):

```diff
diff --git a/indent-fail.scss b/indent-pass.scss
index 5b4d9e4..0a9ae5b 100644
--- a/indent-fail.scss
+++ b/indent-pass.scss
@@ -1,4 +1,4 @@
// @import 'anything';

// Colors variable via accoutrement-color
$colors: (
```

Thanks.

**Comment by DanPurdy** [comment]
@cappadona thanks for the detailed report, 1.8.0 saw massive changes to actually make indentation work for a lot more scenarios and Sass syntax so I think there's obviously still a few edge cases that were missed. I'll look into this one soon. 

👍
